WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -1152,7 +1152,6 @@ enum WooAnalyticsStat: String {
     case loginWooCommerceSetupCompleted = "login_woocommerce_setup_completed"
 
     // MARK: Payments Menu
-    case paymentsMenuCollectPaymentTapped = "payments_hub_collect_payment_tapped"
```

Comment:
After or right before the PR is merged, this event can be marked as deprecated in Tracks.